### PR TITLE
Docs: add WebXR-emulator-extension tools, and clean up.

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

File: docs/page.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ function onDocumentLoad( event ) {
 	text = text.replace( /\[param:([\w\.]+) ([\w\.\s]+)\]/gi, "$2 : <a class=\"param\" onclick=\"window.parent.setUrlFragment('$1')\">$1</a>" ); // [param:name title]
 
 	text = text.replace( /\[link:([\w|\:|\/|\.|\-|\_]+)\]/gi, "[link:$1 $1]" ); // [link:url] to [link:url title]
-	text = text.replace( /\[link:([\w|\:|\/|\.|\-|\_|\(|\)|\#|\=]+) ([\w|\:|\/|\.|\-|\_|\s]+)\]/gi, "<a href=\"$1\"  target=\"_blank\">$2</a>" ); // [link:url title]
+	text = text.replace( /\[link:([\w|\:|\/|\.|\-|\_|\(|\)|\?|\#|\=]+) ([\w|\:|\/|\.|\-|\_|\s]+)\]/gi, "<a href=\"$1\"  target=\"_blank\">$2</a>" ); // [link:url title]
```
Comment (Mugen87) [inline docs/page.js:71]: Do you mind explaining the motivation of this change?

Comment (linbingquan) [reply docs/page.js:71]: I added `|\?|` for `https://marketplace.visualstudio.com/items?itemName=bierner.comment-tagged-templates` and tested it was work, I just tested several links in `Useful-links.html`.

I not sure it has other best ways to do, or this change will effect other links.

**Comment by Mugen87** [comment]
Can you please also update the Chinese version?

https://raw.githack.com/linbingquan/three.js/dev-tools/docs/index.html#manual/zh/introduction/Useful-links

**Comment by linbingquan** [comment]
> Can you please also update the Chinese version?

I have done.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**